packages/form-js-editor/src/render/components/properties-panel/entries/DefaultValueEntry.js

```diff
@@ -56,11 +57,13 @@ export default function DefaultValueEntry(props) {
   if (type === 'radio' || type === 'select') {
     entries.push({
       ...defaultOptions,
-      component: DefaultValueMulti,
+      component: DefaultValueSingleSelect,
       isEdited: isSelectEntryEdited
     });
   }
 
+  // Todo: implement a multiselect equivalent (https://github.com/bpmn-io/form-js/issues/265)
```

Comment:
As a good practice, add your name to the note so the next person can get back to you for more context 👍 
```suggestion
  // todo(Skaiir): implement a multiselect equivalent (cf. https://github.com/bpmn-io/form-js/issues/265)
```